src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <div class="card__top">
+        <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__top-image"
+        >
```

Comment:
I want to ask about previous comment:

![image](https://github.com/mate-academy/layout_product-cards/assets/74732843/d403cac7-e8f9-4202-ba6c-1f9decd21eda)

When previous mentor said about that
![image](https://github.com/mate-academy/layout_product-cards/assets/74732843/cc582ff4-6de1-4403-bda8-88909861964d)

I thought that he says about not hardcoding width OR height. And he was right, when i removed height setting, nothing really changed, so this setting was meaningless. But when i am removing width, my card border spreads over whole page, and the way i could solve this is to set additional margins for my card or padding for entire page, but it looks more like creating problems rather than solving the task. (By the way, when our course mentor Anna was solving this task online, she just set width of card to 200px, just as i did before.)

Am i getting something wrong?